internal/active.go

```diff
@@ -26,6 +26,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/Naman1997/discovr/verbose"
```

Comment:
This file still imports "log" and "fmt" packages. Make sure only the verbose package is used, otherwise it means that we have missed some lines where we are logging stuff.